dialect/psql/starters.go

```diff
@@ -62,18 +62,30 @@ func Arg(args ...any) Expression {
 	return bmod.Arg(args...)
 }
 
+func ArgNamed(names ...string) Expression {
+	return bmod.ArgNamed(names...)
+}
```

Comment:
In the discussion you talked about calling this `NamedArg`, but there is a `NamedArg` function already in this scope, maybe I could call that one `Named`, which would more or less resemble the `sql.Named` function which is for named arguments also?